homework-submission/week1-hw/my-app/src/Post.js

```diff
@@ -0,0 +1,16 @@
+import React, { Component } from 'react';
+
+class Post extends Component {
+
+   render() {
+      return (
+         <div className="todo-item">
+            <div>{this.props.checkbox}</div>
+            <h2> {this.props.description}  </h2>
+            <p>{this.props.deadline}</p>
+         </div>
+      );
```

Comment:
if you do like:
items.map((item, key) => (
                  <Post checkbox={item.checkbox}
                     description={item.description}
                     key={key}
                     deadline={item.deadline}>
                  </Post>
               ))
then you will pass the error in the chrome inspector